src/gui/optionsdialog.ui

```diff
@@ -824,6 +848,9 @@
             </item>
             <item>
              <widget class="QCheckBox" name="checkPreallocateAll">
+              <property name="toolTip">
+               <string>When a torrent is downloaded, create file(s) with file sizes equivalent to the files within the torrent</string>
```

Comment:
maybe: "When a torrent is downloaded, create file(s) with full file sizes to minimize fragmentation"?

Reply:
Alright, done.